instrumentation/message-handler/library/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+plugins {
+  id("otel.library-instrumentation")
+}
+
+dependencies {
+  api(project(":instrumentation:aws-lambda:aws-lambda-core-1.0:library"))
+
+  compileOnly("io.opentelemetry:opentelemetry-sdk")
+
+  library("com.amazonaws:aws-lambda-java-core:1.0.0")
+  library("com.amazonaws:aws-lambda-java-events:2.2.1")
```

Comment:
The AWS-specific parts should maybe be split into a separate library to keep dependencies minimal, also considering Muzzle.

Reply:
I moved the AWS part into a separate library.

"Otherwise, I think there is no use case for a second AWS Lambda specific SQS handler support API."
This new library is not Lambda specific like aws-lamba is. We do not use the Context variable.

I'm not sure if it makes sense to combine message-handler with aws-lambda.

This module is for a fan-in case (many to 1) and is generic and the other library is for a for-loop case (1 to 1) and meant to be used in Lambda layers.

The for-loop case will want to add message ids from the SQS messages for example, where it does not make sense here.

Reply:
> I'm not sure if it makes sense to combine message-handler with aws-lambda.

I mean the SQS message handler, since you are using the types from the `com.amazonaws:aws-lambda-java-events` library. These SQS record types are only used by AWS Lambda, the AWS SDK has a different type for receiving messages. The generic part is a another matter.

Reply:
Well, that's embarrassing.
Thanks for the catch!
I have updated it to use the AWS SDK message objects.